code/datums/logging/_log_entry.dm

```diff
@@ -0,0 +1,177 @@
+/datum/log_entry
+	/*
+	 * The version of this log entry, based on semver: https://semver.org/
+	 *
+	 * This is used to guarantee to automated tooling that the data it is parsing
+	 * is what it is expecting.
+	 *
+	 * For backwards incompatible changes, such as REMOVING fields, or CHANGING their meaning,
+	 * update the MAJOR version number. For instance, 1.2.3 -> 2.0.0.
+	 *
+	 * For backwards compatible additions, such as ADDING fields,
+	 * update the MINOR version number. For instance, 1.2.3 -> 1.3.0.
+	 *
+	 * Otherwise, if you are changing the datum AT ALL, such as to refactor its code or fix an edge case,
+	 * update the PATCH version number. For instance, 1.2.3 -> 1.2.4.
+	 * This is so that in the event your code breaks the logging, automated tooling can isolate it.ted for.
+	 */
+	VAR_PRIVATE/version = "0.0.0"
+	/// The unix timestamp this log entry was created.
+	VAR_PRIVATE/unix_timestamp = 0
+	/// The world.time this log entry was created.
+	VAR_PRIVATE/world_timestamp = 0
+	/// The round this log entry was created on.
+	VAR_PRIVATE/round_id = 0
+	/// The server this log entry was created on.
+	VAR_PRIVATE/server_name
+
+	// required settings
```

Comment:
Spitballing here since I don't know how much other people care, but it's nice when you're able to hover over a variable and this sort of information is present in the DMDoc. For example, you would write it as:
```dm
	/// The category this log entry falls under. Required.
	var/category

	/// The name of the datum that created this log entry. Optional.
	var/source_name

```

Just so that it's clear from the "hover", and you don't have to keep going back to the implementation to see what's mandatory and what isn't.